From 8a6f545f75cd8507b5c59dd799f799d80e298902 Mon Sep 17 00:00:00 2001
From: Jason Smith <jsmith@habana.ai>
Date: Thu, 29 May 2025 08:47:01 -0400
Subject: [PATCH] [SW-230673] agent: enable ascii aux names

Enable kconfig that changes the agent PDR database from using
UTF16 to ASCII strings for storing the various aux name strings.

Change-Id: I9ce98e0043304096b62eee041bfc25dfc65ecf87
Signed-off-by: Jason Smith <jsmith@habana.ai>
---

diff --git a/boards/hl/gaudi3/hl_gaudi3_agent_asic_defconfig b/boards/hl/gaudi3/hl_gaudi3_agent_asic_defconfig
index ffc5c8d..547f93b 100644
--- a/boards/hl/gaudi3/hl_gaudi3_agent_asic_defconfig
+++ b/boards/hl/gaudi3/hl_gaudi3_agent_asic_defconfig
@@ -82,6 +82,7 @@
 CONFIG_LIBPLDM_MAX_MSG_SIZE=1024
 CONFIG_LIBPLDM_RECV_TIMEOUT_MS=10
 CONFIG_LIBPLDM_FIRMWARE_UPDATE=y
+CONFIG_LIBPLDM_PDR_ASCII_AUX_NAMES=y
 CONFIG_BOOTINFO_OOBI_SUPPORT=y
 CONFIG_HL_RESET_DRIVER=y
 CONFIG_THREAD_NAME=y
